modules/metering-use-mysql-or-postgresql-for-hive.adoc

```diff
@@ -9,6 +9,24 @@ The default installation of metering configures Hive to use an embedded Java dat
 
 There are 4 configuration options you can use to control the database used by Hive metastore: url, driver, username, and password.
 
+
+You will create your MySQL or Postgres instance with a username and password. Then either in the OpenShift CLI or by making a yaml file, you will make a secret. The `secretName` you use for this secret needs to map to your MeteringConfig `spec.hive.spec.config.db.secretName`.
+
+To create a secret in Openshift CLI you can use the following command:
+[source,bash]
+$ oc --namespace <YOUR_NAMESPACE> create secret generic <YOUR_SECRETNAME> --from-literal=username=<YOUR_DATABASE_USERNAME> --from-literal=password=<YOUR_DATABASE_PASSWORD>
```

Comment:
We should replace `<YOUR_NAMESPACE>` with `openshift-metering` as we consistently use that namespace (and recommend using that namespace) throughout the example documentation.

Reply:
You can remove `[source,bash]` from above the example. **Note from docs guidelines:** Do not use [source, bash] for oc commands or any terminal commands, because it does not render properly in some cases.

Put `----` on the lines above and below the code block.
https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#code-blocks